src/System.Runtime.Extensions/src/System/Environment.Unix.cs

```diff
@@ -36,6 +37,7 @@ public static partial class Environment
             }
             return results;
         });
+        private static readonly bool IsMac = Interop.Sys.GetUnixName() == "OSX";
```

Comment:
I'd rather not take an avoidable dependency unless there is strong value- we can leave a comment here to use IsPlatform if we ever do take it